Packages/vcs/Lib/vtk_ui/behaviors/draggable.py

```diff
@@ -7,7 +9,8 @@ def __init__(self):
         self.drag_origin = None
         self.drag_position = None
         self.drag_started = None
-        self.drag_buffer = 10 # in pixels
+        self.drag_buffer = 10  # in pixels
+        self.drag_interval = datetime.timedelta(0, .25)
```

Comment:
I realized that this value could be cached and I didn't need quite as many repeated characters/allocations as I was using.